ptp-add-filter-all-releases-anut.js

```diff
@@ -268,6 +269,47 @@
     const get_torrent_objs = async (tracker, html) => {
         let torrent_objs = [];
 
+        if (tracker === "PxHD") {
```

Comment:
As this section was added as an `if`, the following block `HDB` needs to now become `else if`